src/main/java/pl/tomaszdziurko/jvm_bloggers/UTMParametrsConstants.java

```diff
@@ -0,0 +1,25 @@
+package pl.tomaszdziurko.jvm_bloggers;
+
+/**
+ * UTM (Urchin Traffic Monitor) URL query parameter identifiers.
+ * 
+ * @author cslysy <jakub.sprega@gmail.com>
+ */
+public class UTMParametrsConstants {
+    
+    /**
+     * Name of the source (usually the domain of source website).
```

Comment:
I agree, but we should link some reliable source like wikipedia. For me using @see is ok.